bingo.js

```diff
@@ -642,10 +843,19 @@ function copySeedToClipboard(id, event)
 	}
 }
 
-function showCopiedTooltip(event)
+function showCopiedTooltip(event, isAsFixed = false)
 {
-	const x = event.target.offsetLeft + event.target.offsetWidth;
-	const y = event.target.offsetTop;
+	var x = 0;
+	var y = 0;
+	if (isAsFixed) {
+		$("#copiedTooltip").addClass('fixed');
+		x = event.target.getBoundingClientRect().x + event.target.getBoundingClientRect().width;
+		y = event.target.getBoundingClientRect().y;
+	} else {
+		$("#copiedTooltip").removeClass('fixed');
+		x = event.target.offsetLeft + event.target.offsetWidth;
+		y = event.target.offsetTop;
```

Comment:
Found a bug:

![broken-tooltip-Screenshot_20210413_002157](https://user-images.githubusercontent.com/624072/114470053-82911500-9bee-11eb-80d3-a36d2047fd54.png)

It seems that with the jQuery solution from #116, parameter `isAsFixed` is not needed.